.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Don't allow people to merge changes to these generated files, because the result
+# may be invalid.  You need to run "npm install" again.
+package-lock.json               merge=binary
+
```

Comment:
What's this for?

Reply:
Context: I copied what is useful for us here from https://github.com/Azure/azure-sdk-for-js/blob/master/.gitattributes. I needed the `* text=auto eol=lf` line so we do not get conflicts regarding different new lines.

This line is to disable auto-merging of `package-lock.json`